### ISSUE TITLE
Consider React-toastify for simple message-to-user feedback

### ISSUE THREAD

**Issue by CocoisBuggy** [OpenBeta/open-tacos#504]
https://github.com/fkhadra/react-toastify#readme

This is one of my favorite react libraries, I'm gonna punt it here really quickly:

Instead of saying `window.alert` to deliver arbitrary context-divorced messages  to users, you can do

```js
toast.info('message')
toast.error('message')
toast.success('message')
```

I've found in the past that this is a SUPER easy way to show the user feedback in cases where you aren't 100% sure what to do for feedback. You can pretty much sub out all your `console.error` calls and actually SHOW the user - without implementing complex error state or having to worry at all about mobile vs desktop.

It also helps that you can throw the feedback from inside hooks or buried deep in some client-side script

That's my pitch done

**Comment by CocoisBuggy** [comment]
Radix actually supports this kind of thing. I'll write up a demo for what I've described above and report back

**Comment by CocoisBuggy** [comment]
Here is how a super simple toast lifecycle for this kind of problem would look: #510

**Comment by vnugent** [comment]
did you forget to attach a screenshot?

**Comment by CocoisBuggy** [comment]
> did you forget to attach a screenshot?

I meant code-wise, the PR is an example of an event-driven global toast API. react-toastify is a little beefy in terms of bundle size so I was hoping to cut down the size drastically.

It's pretty safe code, I'm just not sure if we actually want to make use of notification-like messages for user feedback. Like anything I guess it's all a balance of pros and cons

**Comment by vnugent** [comment]
Since we're slowly adopting Radix UI primitives, what are some benefits of react-toastify comparing to Radix [Toast](https://www.radix-ui.com/docs/primitives/components/toast#toast)

**Comment by vnugent** [comment]
Fixed in #641
